java/vector/src/main/java/org/apache/arrow/vector/ipc/message/MessageSerializer.java

```diff
@@ -377,8 +371,8 @@ public static ArrowDictionaryBatch deserializeDictionaryBatch(ReadChannel in,
     return new ArrowDictionaryBatch(dictionaryBatchFB.id(), recordBatch);
   }
 
-  public static ArrowMessage deserializeMessageBatch(ReadChannel in, BufferAllocator alloc) throws IOException {
-    Message message = deserializeMessage(in);
+  public static ArrowMessage deserializeMessageBatch(MessageReader reader, BufferAllocator alloc) throws IOException {
```

Comment:
The word "Batch" in the function name is a bit unintuitive. I kind of feel "Message" is a better term than "MessageBatch".

Should we maybe rename this to `deserializeMessage`? 

Also, this message doesn't seem to exclude schema message explicitly. Which also feels a bit weird.

Reply:
Yeah, I think it's ok as is but this seems to be used only in a test.  How about we do a followup PR to refine these functions and we can discuss there?

Reply:
Ok.